UT4MasterServer/Enums/StatisticWindow.cs

```diff
@@ -0,0 +1,9 @@
+﻿namespace UT4MasterServer.Enums;
+
+public enum StatisticWindow
```

Comment:
```suggestion
public enum StatisticWindow : int
```
if this is being used as an integer representing number of days, it could inherit from int

Reply:
Every enum is of int type by default, no need to mention it.